src/language-usability.bib

```diff
@@ -0,0 +1,86 @@
+@inproceedings{brown1991zeus,
+  title={Zeus: A system for algorithm animation and multi-view editing},
+  author={Brown, Marc H},
+  booktitle={Proceedings 1991 IEEE Workshop on Visual Languages},
+  pages={4--9},
+  year={1991},
+  organization={IEEE}
+}
+
+@article{stasko1993methodology,
+  title={A methodology for building application-specific visualizations of parallel programs},
+  author={Stasko, John T and Kraemer, Eileen},
+  journal={Journal of parallel and distributed computing},
+  volume={18},
+  number={2},
+  pages={258--264},
+  year={1993},
+  publisher={Elsevier}
+}
+
+@article{price1993principled,
+  title={A principled taxonomy of software visualization},
+  author={Price, Blaine A and Baecker, Ronald M and Small, Ian S},
+  journal={Journal of Visual Languages \& Computing},
+  volume={4},
+  number={3},
+  pages={211--266},
+  year={1993},
+  publisher={Elsevier}
+}
+
+@article{sorva2013review,
+  title={A review of generic program visualization systems for introductory programming education},
+  author={Sorva, Juha and Karavirta, Ville and Malmi, Lauri},
+  journal={ACM Transactions on Computing Education (TOCE)},
+  volume={13},
+  number={4},
+  pages={15},
+  year={2013},
+  publisher={ACM}
+}
+
+@inproceedings{maletic2002task,
+  title={A task oriented view of software visualization},
+  author={Maletic, Jonathan I and Marcus, Andrian and Collard, Michael L},
+  booktitle={Proceedings First International Workshop on Visualizing Software for Understanding and Analysis},
+  pages={32--40},
+  year={2002},
+  organization={IEEE}
+}
+
+@inproceedings{murphy2010interactive,
+  title={An interactive ambient visualization for code smells},
+  author={Murphy-Hill, Emerson and Black, Andrew P},
+  booktitle={Proceedings of the 5th international symposium on Software visualization},
+  pages={5--14},
+  year={2010},
+  organization={ACM}
+}
+
+@inproceedings{reiss2005paradox,
+  title={The paradox of software visualization},
+  author={Reiss, Steven P},
+  booktitle={3rd IEEE International Workshop on Visualizing Software for Understanding and Analysis},
+  pages={1--5},
+  year={2005},
+  organization={IEEE}
+}
+
+@incollection{de2002visualizing,
+  title={Visualizing the execution of Java programs},
```

Comment:
proper nouns in Bibtex titles need to be in curly braces, i.e. `{Java}`, so that the capitalization works correctly